source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards.xcodeproj/project.pbxproj

```diff
@@ -805,7 +805,6 @@
 				6BBE841823CD184D00ECA586 /* ACOWarning.h */,
 				6BBE841723CD184D00ECA586 /* ACOWarning.mm */,
 				6B1147D01F32E53A008846EC /* ACRActionDelegate.h */,
-				6B616C4121CB20D1003E29CE /* ACRActionToggleVisibilityRenderer.h */,
```

Comment:
>ACRActionToggleVisibilityRenderer [](start = 32, length = 33)

Is this a breaking change ?

Reply:
No, I just re-ordered file in Xcode project.